cli/kaos_cli/facades/backend_facade.py

```diff
@@ -124,6 +210,44 @@ def _tf_init(self, provider, env, local_backend, destroying=False):
             self.tf_service.init(directory)
         return directory
 
+    def _set_context_list(self, current_context):
+        try:
+            available_contexts = self.state_service.get(CONTEXTS, 'environments')
+        except KeyError:
+            available_contexts = ''
+            
+        updated_contexts = available_contexts + current_context if not available_contexts else \
+            available_contexts + ',' + current_context
+        self.state_service.set(CONTEXTS, environments=updated_contexts)
+
+    def _unset_context_list(self, current_context):
+        available_contexts = self.state_service.get(CONTEXTS, 'environments')
+        context_list = available_contexts.split(',')
+        if len(context_list) <= 1:
+            updated_context_list = []
+        else:
+            try:
+                context_list.remove(current_context)
+            except ValueError:
+                updated_context_list = context_list
```

Comment:
I would simplify this block by assigning `updated_context_list` to an empty list and only access the try/except block as `if context_list:`.

Reply:
This has been re-written differently as well in order to handle empty values